webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index: './src/index.js',
+    print: './src/print.js',
+  },
+  devtool: 'inline-source-map',
+  devServer: {
+    contentBase: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      title: 'To Do List',
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+};
```

Comment:
I'm facing an error with your webpack setup.
Isn't possible, run the server. Please fix this issue:
![Screen Shot 2021-08-19 at 19 44 32](https://user-images.githubusercontent.com/22258274/130161795-6a6bdd96-545e-40d4-affa-d63feefda145.png)